show/main.py

```diff
@@ -187,6 +186,38 @@ def summary(interfacename):
         command = cmd_ifconfig
         run_command(command)
 
+
+@interfaces.group(cls=AliasedGroup, default_if_no_args=True)
+def transceiver():
+    pass
+
+interfaces.add_command(transceiver)
+
+@transceiver.command(default=True)
+@click.argument('interfacename', required=False)
+def default(interfacename):
+    if interfacename is not None:
+        command = "sudo sfputil -p {}".format(interfacename)
```

Comment:
`sfputil` syntax changed recently. Please update all `sfputil` calls here and below to new syntax.

Reply:
done